src/ralph/accounts/admin.py

```diff
@@ -208,6 +209,19 @@ def url(self, item):
     url.title = _('Link')
 
 
+class AssignedSimcardsList(Table):
+
+    def url(self, item):
```

Comment:
Regular users with regular permissions won't be able to go any of their sim cards. It's basically the same issue as happens when someone clicks on "go to asset" from My Equipment. I would suggest to remove the link.

P. S. It's fine, If removing the link here will also remove the "go to asset link".

Reply:
I removed url. It's not necessary in this case.